ww4ff/align/vad.py

```diff
@@ -16,14 +16,14 @@ def detect_vad(frames):
             start = 0
             vad = webrtcvad.Vad(3)
             for frame in frames:
-                buf = (frame.numpy() * 32767 / 1.414)
+                buf = frame.numpy() * 32767
                 buf = buf.astype(np.int16).tobytes()
                 if frame.size(0) < frame_len or vad.is_speech(buf, audio.sample_rate):
                     break
                 start += self.frame_ms
             return start
         from ww4ff.data.dataset import AudioClipExample
-        audio = audio  # type: AudioClipExample
+        audio  # type: AudioClipExample
```

Comment:
have 
```
    def align(self, audio: AudioClipExample) -> AlignedTranscription:
```
and remove this line?

Reply:
I had that originally, but that results in circular import.

Reply:
hrm okay. let's get back to this